src/sdgio-edit.owl

```diff
@@ -1515,6 +1519,24 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020232> "system level outcome"@en)
 SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020232> <http://purl.unep.org/sdg/SDGIO_00000251>)
 
+# Class: <http://purl.unep.org/sdg/SDGIO_00020236> (intermediate development outcome)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020236> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020236> "2019-04-28T11:19:21Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.unep.org/sdg/SDGIO_00020236> "IDO")
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) rdfs:comment <http://purl.unep.org/sdg/SDGIO_00020236> "A CGIAR objective specification which 1) names an objective the achievement of which contributes to the achievement of a specific system level outcome and 2) indicates to which system level outcome a piece of CGIAR research contributes and how such a piece of research contributes to that system level outcome.")
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020236> "intermediate development outcome"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00000251>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020237> (sub-intermediate development outcome)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020237> "A CGIAR objective specification which 1) names an objective the achievement of which contributes to the achievement of an IDO and 2) names an objective of a specific and limited activity of researchers and policy makers.")
```

Comment:
Similar definitional issues to the IDO def. Suggested rephrasing:

```
A CGIAR objective specification which describes the intended endpoint and outcome(s) of a process which, itself, is part of and contributes to a larger process directed by an intermediate level outcome.
```

Reply:
I de-loaded the definition of sub-IDO. However, as before, I ask you to confirm suggestion of saying that a sub-IDO _describes_ something rather than _refers_ (or names or something like that) to something.

Reply:
In the end, I accepted Pier Luigi's suggestion, as I understood that he repeated it on Skype. We will talk about this on 29 Apr 2019.

Reply:
I am going to resolve this conversation.
Summary of the issue: the definition of the sub-IDO class was too complicated and did not align with conceptual rules states in IAO.
Solution: I modified the definition according to Pier Luigi's suggestion.